src/converter.ts

```diff
@@ -531,12 +559,19 @@ export class ProtocolToMonacoConverter {
         }
     }
 
-    asCompletionResult(result: CompletionItem[] | CompletionList | undefined): monaco.languages.CompletionItem[] | monaco.languages.CompletionList | undefined {
+    asCompletionResult(result: CompletionItem[] | CompletionList | undefined): monaco.languages.CompletionList {
         if (!result) {
-            return undefined;
+            return {
+                isIncomplete: false,
+                items: []
+            }
         }
         if (Array.isArray(result)) {
-            return result.map(item => this.asCompletionItem(item));
+            const items = result.map(item => this.asCompletionItem(item));
+            return {
+                isIncomplete: false,
+                items: items
+            }
         }
         return <monaco.languages.CompletionList>{
```

Comment:
fyi: `<monaco.languages.CompletionList>` is redundant, if a return type only `monaco.languages.CompletionList`

Reply:
Should've removed this, thanks for catching this!